src/porepy/viz/exporter.py

```diff
@@ -520,9 +518,44 @@ def _from_vector_format(
                             )
 
                             offset += intf.num_cells
+                if key in vtu_data.point_data:
+                    value = vtu_data.point_data[key]
```

Comment:
Just double-checking. No concatenating needed here?

Reply:
I assume as there no different "point-types", there may be no need?

Reply:
correct, that was the reason. for the cell data the code subdivides depending on the type of cell, with the points no